_posts/2021-03-14-curating/curating.Rmd

```diff
@@ -0,0 +1,329 @@
+ [getchirrapp.com](https://getchirrapp.com/){target="_blank"} to me the week of my curation. I also kept [emojipedia](https://emojipedia.org/){target="_blank"} open all week
+ to easily copy and paste emojis into drafts.
+ 
+Not all content was premeditated - I also tweeted in the moment. For example, the [W.E.B. Du Bois'](https://twitter.com/WeAreRLadies/status/1362016896116219904?s=20){target="_blank"} `#TidyTuesday` visualizations were incredible that week, or when I [realized](https://twitter.com/WeAreRLadies/status/1362509983368249346?s=20){target="_blank"} a new colleague wasn't yet taking advantage of RStudio projects.
+
+# Content inspiration
+
+As I approached my curating week, I recalled previous `@WeAreRLadies` that were
+memorable for me, my previous experience as an educator, and some reflection
+questions to inspire content.
+
+| Inspiration  source | Example realization   |
+|---------------------|-----------------------|
+| 1. Rotating curator Mine Çetinkaya-Rundel ([`@minebocek`](https://twitter.com/minebocek){target="_blank"}) tweets awesome [gifs](https://twitter.com/WeAreRLadies/status/1064704918102163457){target="_blank"}| Gifs for R code demos like [this one](https://twitter.com/WeAreRLadies/status/1361802517735178243){target="_blank"}  |
+| 2. Rotating curator Megan Stodel ([`@MeganStodel`](https://twitter.com/MeganStodel){target="_blank"}) tweets a [project](https://twitter.com/WeAreRLadies/status/1313177623128944645){target="_blank"} inspired by curating | An R project to [introduce myself](https://twitter.com/WeAreRLadies/status/1361286341317779456){target="_blank"} as a curator |
+| 3. Rotating curator Julia Piaskowski ([`@SeedsAndBreeds`](https://twitter.com/SeedsAndBreeds){target="_blank"}) tweets a great technical thread on [ANOVA](https://twitter.com/WeAreRLadies/status/1223790298024726528){target="_blank"} | A thread on [blogging resources](https://twitter.com/WeAreRLadies/status/1363144545677017089){target="_blank"} |
+| 4. Prior experience as an educator | Starting discussion with a [question](https://twitter.com/WeAreRLadies/status/1361332603274612739){target="_blank"} |
+| 5. What am I passionate about lately? | [Blogging](https://twitter.com/WeAreRLadies/status/1363144545677017089){target="_blank"} |
+| 6. What did I have to overcome to be where I am today? | Learning how to [ask for help online](https://twitter.com/WeAreRLadies/status/1362370580708790274){target="_blank"} |
+| 7. What have colleagues or students asked me about? | What needs [updating](https://twitter.com/WeAreRLadies/status/1362114431090573315){target="_blank"}  and when |
+| 8. What are some R functions or packages that have helped me recently? | [sortable ](https://twitter.com/WeAreRLadies/status/1361418527870132226){target="_blank"} package |
+| 9. What are R-Ladies voices I can amplify while I have this large platform? | Quote tweeting questions like [this](https://twitter.com/WeAreRLadies/status/1362866281918128132){target="_blank"} |
```

Comment:
```suggestion
| 9. What are R-Ladies voices I can amplify while I have this large platform? | [Quote tweeting questions](https://twitter.com/WeAreRLadies/status/1362866281918128132){target="_blank"} |
```

I assume "this" links are best avoided like "here" links cf https://webaccess.berkeley.edu/ask-pecan/click-here (that someone told me about on GitHub a while back actually!).